modules/ocf_desktop/manifests/xsession.pp

```diff
@@ -129,14 +129,26 @@
   }
 
   # use ocf logo on login screen
-  file {
-    ['/usr/share/icons/Adwaita', '/usr/share/icons/Adwaita/256x256', '/usr/share/icons/Adwaita/256x256/status']:
-      ensure => directory;
-    '/usr/share/icons/Adwaita/256x256/status/avatar-default.png':
-      ensure  => link,
-      target  => '/opt/share/xsession/images/ocf-color-256.png',
-      require => File['/opt/share/xsession/images'];
-  }
+  if $::lsbdistcodename == 'bullseye' {
+    file {
+      ['/usr/share/icons/Adwaita', '/usr/share/icons/Adwaita/512x512', '/usr/share/icons/Adwaita/512x512/status']:
+        ensure => directory;
+      '/usr/share/icons/Adwaita/512x512/status/avatar-default.png':
+        ensure  => link,
+        target  => '/opt/share/xsession/images/ocf-color-512.png',
```

Comment:
This is an image we provide AIUI. So I think this should stay 256 and not change to 512?